src/crypto.rs

```diff
@@ -1,28 +1,31 @@
 use tiny_keccak::Keccak;
-use hash::H256;
+use types::H256;
 
 /// Compute keccak hash.
 ///
 /// # Examples
 ///
 /// ```rust
-/// use pwasm_std::keccak;
+/// use pwasm_std::{keccak, types};
 ///
-/// let hash = keccak("hello world");
+/// let hash = keccak("hello world".as_bytes());
 ///
-/// let expected = &[
+/// let expected = types::H256::from([
 /// 	71, 23, 50, 133, 168, 215, 52, 30,
 /// 	94, 151, 47, 198, 119, 40, 99, 132,
 /// 	248, 2, 248, 239, 66, 165, 236, 95,
 /// 	3, 187, 250, 37, 76, 176, 31, 173
-/// ];
+/// ]);
 ///
-/// assert_eq!(hash.as_ref(), expected);
+/// assert_eq!(hash, expected);
 /// ```
-pub fn keccak<T>(input: T) -> H256 where T: AsRef<[u8]> {
+pub fn keccak<T>(input: T) -> H256
+where
```

Comment:
hmm.. I wonder, is that ok indentation?..

Reply:
It is standard Rust indentation convention but maybe against parity rules - I have to check that.